starter-code/scripts/article.js

```diff
@@ -13,18 +19,31 @@ Article.prototype.toHtml = function() {
   However, in our modules.css stylesheet, we gave all elements
   with a class of template a display of none. Let's make
   sure we're not accidentally hiding our cloned article! */
+  // estimated time: 20 min, actual time: 10 min
+
+  $newArticle.removeClass('template');
 
   if (!this.publishedOn) $newArticle.addClass('draft');
   $newArticle.data('category', this.category);
-
   /* TODO: Now use jQuery traversal and setter methods to fill in the rest
   of the current template clone with properties from this particular Article instance.
   We need to fill in:
-    1. author name,
-    2. author url,
-    3. article title,
-    4. article body, and
-    5. publication date. */
+  1. author name,
+  2. author url,
+  3. article title,
+  4. article body, and
+  5. publication date. */
+
+  // Estimated time: 30 min, actual time 50 min:
+
+  $newArticle.find('a').text(this.author);
+  $newArticle.find('a').text(this.url).attr('href', this.authorUrl);
```

Comment:
I do not think you have a this.url. You can just target the tag and then add an attribute!